docs/Concepts/4.md

```diff
@@ -1,60 +1,104 @@
-These features are provided for in [WAKU-MESSAGE (version 1)](https://rfc.vac.dev/spec/14#version-1) and [WAKU-NOISE](https://rfc.vac.dev/spec/35/) through payload encryption as well as encrypted signatures.
+## Use your own nodes
 
-> `WAKU-NOISE` enables secure channel negotiation over Waku.
+Developers have the choice to run their own [nwaku](https://github.com/status-im/nim-waku/) nodes. Read [Nwaku Service Node](https://github.com/status-im/nwaku/tree/master/docs/operators) to learn how to run your own node.
 
-## Security Considerations
+Once you have started your nwaku node(s), you could decide to pass a list of predefined nodes as a way to bootstrap your own node.
 
-**Lack of anonymity/unlinkability in the protocols involving direct connections including [`13/WAKU2-STORE`](https://rfc.vac.dev/spec/13/) and [`12/WAKU2-FILTER`](https://rfc.vac.dev/spec/12/) protocols**:
-The anonymity/unlinkability is not guaranteed in the protocols like `13/WAKU2-STORE` and `12/WAKU2-FILTER` where peers need to have direct connections to benefit from the designated service.
-This is because during the direct connections peers utilize `PeerID` to identify each other,
-therefore the service obtained in the protocol is linkable to the beneficiary's `PeerID` (which counts as PII).
-For `13/WAKU2-STORE`, the queried node would be able to link the querying node's `PeerID` to its queried topics.
-Likewise, in the `12/WAKU2-FILTER`, a full node can link the light node's `PeerID`s to its content filter.
+**Pros:**
+
+- Low latency
+- Low resource requirements
+
+**Cons:**
+
+- Prone to censorship: node IPs can be blocked
+- Limited: Static number of nodes
+- Poor maintainability: Code needs to be changed to update the list
+
+## DNS Discovery
+
+[EIP-1459: Node Discovery via DNS](https://eips.ethereum.org/EIPS/eip-1459) has been implemented in js-waku, nwaku and go-waku with some modification on the [ENR format](https://rfc.vac.dev/spec/31/).
+
+DNS Discovery enables anyone to register an ENR tree in the `TXT` field of a domain name.
+
+_ENR_ is the format used to store node connection details (IP, port, multiaddr, etc).
+
+This enables a separation of software development and operations as dApp developers can include one or several domain names to use for DNS discovery, while operators can handle the update of the dns record.
+
+It also enables more decentralized bootstrapping as anyone can register a domain name and publish it for others to use.
+
+**Pros:**
+
+- Low latency, low resource requirements,
+- Bootstrap list can be updated by editing a domain name: no code change is needed,
+- Can reference to a greater list of nodes by pointing to other domain names in the code or in the ENR tree.
+
+**Cons:**
+
+- Prone to censorship: domain names can be blocked,
+- Limited: Static number of nodes, operators must provide their ENR to the domain owner to get their node listed.
+
+## Other Discovery Mechanisms
```

Comment:
Can be removed for now.

Reply:
Just the cons? Or also the pros?

Reply:
The whole "Other Discovery Mechanisms" section